server/api/cvrs.py

```diff
@@ -1355,23 +1377,18 @@ def validate_cvr_upload(
         raise BadRequest("Invalid file type")
 
     if cvr_file_type == CvrFileType.HART:
+
+        def is_zip_file(file):
+            return file.mimetype in ["application/zip", "application/x-zip-compressed"]
+
         files = request.files.getlist("cvrs")
-        num_zip_files = 0
-        num_csv_files = 0
-        for file in files:
-            if file.mimetype in ["application/zip", "application/x-zip-compressed"]:
-                num_zip_files += 1
-            elif does_file_have_csv_mimetype(file):
-                num_csv_files += 1
-        if num_zip_files == 0:
-            raise BadRequest("Please submit a ZIP file export.")
-        if not (
-            (num_zip_files == len(files))
-            or (num_zip_files == len(files) - 1 and num_csv_files == 1)
```

Comment:
This validation was previously enforcing that there be only one scanned ballot information CSV, a constraint that this PR removes